### PR TITLE
Add retrospective section to Whitepaper README.md

### PR DESCRIPTION
**Pull request by PushkarJ**
Related: #480

### CODE REVIEW

**Comment by PushkarJ** [comment]
Spellcheck is failing for `CNSWP` and `CNSWP's`. Is there a way to add this to allow list?

**Comment by lumjjb** [comment]
Yep - i think you can just add it to https://github.com/cncf/tag-security/blob/main/ci/spelling-config.json

**Comment by PushkarJ** [comment]
@lumjjb ready for review 🤞🏼

File: security-whitepaper/README.md [hunk 1]
```diff
@@ -58,6 +58,21 @@ Upon completion of review, the TAG-Security TOC Liaison shall provide final
 approval on the PR.  At which point the markdown state will be changed to
 "Approved" and merged.
 
+### Retrospective
+
+Release of the whitepaper, triggers a retrospective process that will assess
```
Comment (TheFoxAtWork) [inline security-whitepaper/README.md:63]:
```suggestion
Release of the whitepaper triggers a retrospective process that will assess
```

**Review by TheFoxAtWork — APPROVED** [review]
LGTM just a comment on a comma

**Pull request merged**